app/main.py

```diff
@@ -82,7 +83,9 @@ def create_app(*args, **kwargs):
 def register_extensions(app: flask.Flask) -> None:
     app.secret_key = APP_SESSION_SECRET
 
+    app.config['SQLALCHEMY_ECHO'] = DEBUG
```

Comment:
minor comment, what does `DEBUG` mean here?

Reply:
`SQLALCHEMY_ECHO` option specifies if all the queries issued by the ORM (SQLAlchemy) should be printed to stdout on the server for troubleshooting/analysis purposes. We only want this to happen when in `debug mode`. As we already have such setting (treated as boolean flag, defaults to `False`) it means: if debug is enabled (`True`) also enables `SQLALCHEMY_ECHO`.